templates/error.html.twig

```diff
@@ -0,0 +1,24 @@
+{% extends 'layout.html.twig' %}
+
+{% block content %}
+    <div class="content error-page">
+        <div class="is-center">
+            <h2>Oops</h2>
+
+            <p>{{ exception.message }}</p>
```

Comment:
You display the real error? It can be very technical. Maybe display it in small, but add a real text on top like "Our application was not able to do what you asked for. That's unfortunate and we hope it didn't break Christmas. Feel free to contact us if you need assistance!".

Reply:
This template is uses only to show OUR custom exceptions that are always safe to display :wink:
I will add some text before :+1: